agent/consul/server_log_verification.go

```diff
@@ -62,12 +62,12 @@ func makeLogVerifyReportFn(logger hclog.Logger) verifier.ReportFn {
 			if r.WrittenSum > 0 && r.WrittenSum != r.ExpectedSum {
 				// The failure occurred before the follower wrote to the log so it
 				// must be corrupted in flight from the leader!
-				l2.Info("verification checksum FAILED: in-flight corruption",
```

Comment:
Not for now...but....Would it be helpful to have metrics here for failures?  Or is discovering these via aggregrated logs be enough?

Reply:
We do already - they are emitted by the verifier directly - see the telemetry.mdx additions.